packages/docsearch-css/src/button.css

```diff
@@ -61,6 +61,9 @@
   border: 0;
   top: -1px;
   width: 20px;
+  transition-property: all;
+  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+  transition-duration: 100ms;
```

Comment:
Could we still respect `prefers-reduced-motion` even though the animation is subtile?

Adding this below should work:

```css
@media screen and (prefers-reduced-motion: reduce) {
  .DocSearch-Button-Key {
    transition: none;
  }
}
```